news/2021-12-27-clickers-choice-2021.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "The Best osu! Players of 2021"
+date: 2021-12-27 09:00:00 +0000
+---
+
+It's time for the community to decide the top 20 osu! players of 2021!
+
+![](/wiki/shared/news/2021-12-27-clickers-choice-2021/top_20_players_of_2021_banner.png)
+
+2021 was an incredible year for many circle clickers, some even with record-breaking performances in solo-play and tournaments. Now that the year's coming to an end, we're bringing you the **9th iteration of the international top 20 voting**, courtesy of my latest project: [Clicker's Choice](https://choice.hwc.hr/)!
+
+The Top 20 vote will be run on the Clicker's Choice website, which is already home to the top player voting for Germany, Poland, Canada, UK and many other countries. Clicker's Choice aims to make it easier for users to vote, while also making it simpler for the host to tabulate and evaluate results.
+
+As a voter, it's pretty straightforward — just log in through osu! and start voting! **Everything regarding the criteria and deadlines are on the voting page as well.** Here's a showcase of the entire process:
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://assets.ppy.sh/media/hwchr/clickers_choice_promo.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+If you have any further questions, feel free to contact [hallowatcher](https://osu.ppy.sh/users/1874761) directly.
+
+This needs a better post format beucase it oesn't read well at all.. [TOP 20 VOTING](https://choice.hwc.hr/vote/6)
+
```

Comment:
maybe like this

```suggestion
But other than that, what are you waiting for? [Go ahead and vote!](https://choice.hwc.hr/vote/6)
```

Reply:
Oh woops, I already changed it. what do you think of the version i committed?

Reply:
This reads better considering paragraph flow, and the link doesn't need to include both sentences

Reply:
done